index.js

```diff
@@ -278,19 +280,45 @@ class GithubScm extends Scm {
             "else echo 'sd-step exec core/git'; fi)";
         const command = [];
 
-        // Git clone
-        command.push(`echo Cloning ${checkoutUrl}, on branch ${config.branch}`);
         command.push('if [ ! -z $SCM_CLONE_TYPE ] && [ $SCM_CLONE_TYPE = ssh ]; ' +
             `then export SCM_URL=${sshCheckoutUrl}; ` +
             'elif [ ! -z $SCM_USERNAME ] && [ ! -z $SCM_ACCESS_TOKEN ]; ' +
             `then export SCM_URL=https://$SCM_USERNAME:$SCM_ACCESS_TOKEN@${checkoutUrl}; ` +
             `else export SCM_URL=https://${checkoutUrl}; fi`);
-        command.push(`${gitWrapper} `
-                + `"git clone --recursive --quiet --progress --branch ${config.branch} `
-                + '$SCM_URL $SD_SOURCE_DIR"');
-        // Reset to SHA
-        command.push(`${gitWrapper} "git reset --hard ${checkoutRef} --"`);
-        command.push(`echo Reset to ${checkoutRef}`);
+
+        if (config.manifest) {
+            const repoDownloadUrl = 'https://storage.googleapis.com/git-repo-downloads/repo';
+            const sdRepoReleasesUrl = 'https://github.com/screwdriver-cd/sd-repo/releases/latest';
+
+            command.push('echo Checking out code using the repo manifest defined in '
+                + `${config.manifest}`);
+
+            // Get the repo binary
+            command.push(`curl ${repoDownloadUrl} > /usr/local/bin/repo`);
```

Comment:
This is a python script. Maybe we need to make sure python is setup in the container as well? I'm not quite sure what's the best way to ensure that though

Reply:
As per discussion with Renaud, it seems that requiring images to contain Python to make use of Repo is reasonable.

We should definitely document this and other dependencies as part of the release post.